instrumentation/all/lib/opentelemetry/instrumentation/all.rb

```diff
@@ -11,6 +11,7 @@
 require 'opentelemetry-instrumentation-active_job'
 require 'opentelemetry-instrumentation-active_record'
 require 'opentelemetry-instrumentation-action_view'
+require 'opentelemetry-instrumentation-action_mailer'
```

Comment:
I believe this also needs to be added as a dependency in the "instrumentation-all" gem's gemspec.

Reply:
I saw all gem's gemspec only have `spec.add_dependency 'opentelemetry-instrumentation-rails', '~> 0.30.0'`, without other rails components' instrumentation.